pkgs/development/python-modules/natural/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
```

Comment:
```suggestion
, lib
, setuptools
, six
```

Reply:
thank you for those reviews, I'm gonna fix everything 👍